### ISSUE TITLE
PH Global divisions

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#1296]
As per #1285 @jenniferisasi has raised some concerns about the current planned breakdown of the PH Global roles.

These were:

Head of PH Global (has strategic oversight of PH Global team (leadership))
PH Global - Africa (responsible for promoting the project in a specific region, and listening to the needs of that region to feed them back to the project team)
PH Global - Europe
PH Global - Middle East
PH Global - Asia / Pacific
PH Global - North America
PH Global - Latin America

@jenniferisasi is instead suggesting a linguistic approach.

> I want to raise again the inconvenience of dividing these roles into regions rather than languages as it will not keep things simple as Adam claims. Regions doesn't equate/parallel infrastructural and cultural factors, issues, or privileges; nor does language itself but it facilitates communication and cultural awareness at the minimum. Maybe to make my case more clear I want to ask: why are the US and Canada separated from the UK, Australia, India and the English speaking Caribbean?

> Also, do you (plural) see the convenience and easiness of North America being only the US and Canada with practically the same resources, language and Western culture, versus the difficulty and intricacies of potentially building relationships with 54 countries in Africa with radically different resources, languages and cultures in most cases? That complexity is even more apparent than the one for Latin America.

**Comment by acrymble** [comment]
This was my idea to set up in the first place, so I'll try to explain my thinking.

We have lots of activity in the certain parts of the world, and none at all in others. For example, we've hosted events or given talks in Germany, Switzerland, the UK, Canada, the US, Mexico, and Colombia (off the top of my head). But we have never had anyone presenting the project to audiences in Asia or Australia, as far as I know.

I'd like us to be active in as many different parts of the world as possible, which is why I think it makes sense to give certain people a role trying to think up and execute ways that we can be active in different territories. The reason I don't think it makes sense for someone to be in charge of "English" outreach is because we already have that person, and their role is "Managing Editor". It's also not feasible to have someone trying to organize workshops in England, Canada, and Australia, purely because of the time zones and the cost. I might be able to go to Belgium from London, but I won't go to Sydney.

We have no activity at all in Africa right now. I think it would be a good idea for someone to take it upon themselves to change that. If the activity grows to the point where we need more people taking on that portfolio, great we can revise it. But at the moment I think one person thinking about Africa makes sense, and it's a big improvement on what we have right now.

Exactly what the territories are, I am not that bothered. As long as someone helps us make exciting strides into places we haven't been before.

**Comment by arojascastro** [comment]
I have mixed feelings about this issue. I agree that the division into territories is practical but it may be too simplistic. I think we could just define priorities and actions and then assign roles. I do not think we can keep growing for many reasons, I am very happy with three languages and three editions, but I think there is plenty of room for our French team to find collaborators in French Africa for instance or for finding an editor from India or from any other English-speaking country that is not Canada, the States or the UK or organize and event in one of this countries, or simply recruit more reviewers from neglected areas. 

I would like also to say that the delegate in Europe... well, that's going to be hard for anyone because I cannot see how this person is going to act in such a diverse scenario: I do not see myself exploring what's going on in Russia or in Norway, I have an idea about what's going on in Spain and Germany only. 

I would only have three roles that couple with our editions:

- English role - that is going to try to expand its action in India for instance
- Español role - the same but maybe with a neglected area in Latin America (I do not know which, because we are quite fine, I think)
- Français role - that is going to work on a certain cultural region in Africa or wherever they want

To sum up, I think we should be less ambitious and define priorities first of all, have some discussions and then create those roles (maybe "missions" or "actions" rather than roles). But this is only my uninformed opinion and I may change my mind if I hear more opinions because I do not have a strong opinion on it. I just would like we could relax and get new people comfortable with the current state of the project.

**Comment by vgayolrs** [comment]
This is a truly interesting discussion that shows us the cultural, economic, social and technological problems beyond the divide between the global north and global south.

Let's give an example. Now that I am working on a committee of ADHO 2019, I find that the Utrech conference is focused on Africa: there is a whole program to support the development of the digital humanities in Africa (workshops, special bursaries, spaces dedicated especially for posters on projects from Africa - [see here](https://dh2019.adho.org/programme/focus-on-africa/)). Well, very welcome: Inclusivity is going beyond the barriers of divisions (gender, culture, language, etc.).

But what is Africa? Of all the proposals in a certain field for evaluation (of which I can not speak now until the result is made public), only one proposal came from Africa. Ok, at least we have something. But this proposal comes from South Africa, which is a country with a level of development more "European" than the rest of the African countries.

South Africa speaks English (and other languages of its own), just as in Kenya English and Swahili are spoken; but Mozambique and Angola speak Portuguese and Senegalese speak French. Most of the non-European world is connected, in a cultural sense, to its colonial past. And that colonial past is not homogenous in terms of the classical continental geographical regions marked by Western thought (a white, Anglo-Saxon, European-centered thought). Access to digital technology in India is not the same as in Pakistan or Nepal, three component countries of South Asia. As you know, one of the countries in that region that have the most access to PH is India, where they speak English and Hindi.

In this way, and following the concern of @arojascastro , I would like to point out that the construction of cultural communities has a different logic to the configuration of a geographically structured world in the strict sense of physical regions. Therefore, @jenniferisasi  's criticism is very valuable and we have to understand it in this sense. What connects globally us on the Internet is the language, which is part of the rest of the cultural values, not the geographic location in which we are located. And in PH I think we have the opportunity to make a shift in this matter.

**Comment by acrymble** [comment]
There's a couple of different issues being raised here, but one of them I want to try to address is 

> I would like also to say that the delegate in Europe... well, that's going to be hard for anyone because I cannot see how this person is going to act in such a diverse scenario: I do not see myself exploring what's going on in Russia or in Norway, I have an idea about what's going on in Spain and Germany only.

Of course you can't be an expert on Norway. But what a "European" person can do is point out to an American team member that idea X and solution Y doesn't make sense in the European context. You won't always know, but you'll probably know more often than anyone living in the US, and that's a strength for the project.

It was that approach that led us to take on our first non-North American team members.

**Comment by arojascastro** [comment]
Agreed on *what a "European" person can do is point out to an American team member that idea X and solution Y doesn't make sense in the European context*, but do we need a role for that? Maybe yes to raise awareness. Why we do not do some brain storming about concrete actions that we could develop in 2019?

**Comment by acrymble** [comment]
Sure @arojascastro. Here are a few concrete actions I think we could or should pursue. Please feel free to add to it.

- Foster the community developed in Bogotá and encourage more submissions from participants, as many of these lessons are already half-written. Best achieved by someone Spanish speaking.
- Engage with West Africa or South Africa in a similar way, trying to identify and foster a community of potential contributors, while also learning about the needs in the region and helping us understand how best to meet those. This could be done in either French or English.
- Make connections with our user-base in India. Find out who they are and how we can serve them better. This should be done in English.

**Comment by jenniferisasi** [comment]
Thanks everyone! I like the idea of developing strategies or actions for specific communities rather than trying regions as such. It makes more sense to me and, overall, it will be easier to achieve our goals. 

In this sense, and merging @arojascastro and @acrymble ideas, plus the people who signed for the roles already, it would look something like this: 

- In Spanish: Foster the community developed in Bogotá and encourage more submissions @rivaquiroga (in process)
- In French: Engage with francophone West Africa to learn about their work and resources and foster collaborations that includes them as contributors @spapastamkou (in process with Senegal) 
- In English: Engage with the user-base in India and South Africa to learn about their work and resources and foster collaborations that includes them as contributors @? Maybe @vgayolrs can contact the people working towards DH19 after the even takes place to connect them with us. 

I would like to know their ideas on these.

**Comment by jenniferisasi** [comment]
It is worth checking our analytics not just for one-time users but also the average duration of the session, as it shows (in this map for 2019) very specific places where people are visiting and may be overlooked: 

<img width="903" alt="2019may11_duracionmedia" src="https://user-images.githubusercontent.com/11022533/57583532-be695000-7496-11e9-8c24-5c6b07100f0c.png">

**Comment by acrymble** [comment]
@jenniferisasi I agree with your sentiments shown in the map, but that seems to support my proposal rather than yours.

**Comment by drjwbaker** [comment]
Thanks all for this discussion. Really interesting.

My substantive comment is in support of having PH Global roles that **do not map** with our 3 (current) publications (these don't have to be the roles @acrymble proposed). My worry with - say @arojascastro's proposal at https://github.com/programminghistorian/jekyll/issues/1296#issuecomment-491375947 - is that aligning PH Global with the language publications would inhibit collaboration between team members in PH Global roles. As @jenniferisasi, @arojascastro , and @vgayolrs all imply/note, roles like PH Global North America may be less complex to manage than PH Global Africa or PH Global Europe. If that becomes the case we might than start dividing up the PH Global Africa roles further (PH Africa (Anglophone), PH Africa (Francophone), et cetera). But to do that now seems a little premature to me.

**Comment by arojascastro** [comment]
> It is worth checking our analytics not just for one-time users but also the average duration of the session, as it shows (in this map for 2019) very specific places where people are visiting and may be overlooked:
> 
> <img alt="2019may11_duracionmedia" width="903" src="https://user-images.githubusercontent.com/11022533/57583532-be695000-7496-11e9-8c24-5c6b07100f0c.png">

Can you develop your ideas a little bit more please?

**Comment by jenniferisasi** [comment]
> > It is worth checking our analytics not just for one-time users but also the average duration of the session, as it shows (in this map for 2019) very specific places where people are visiting and may be overlooked:
> > <img alt="2019may11_duracionmedia" width="903" src="https://user-images.githubusercontent.com/11022533/57583532-be695000-7496-11e9-8c24-5c6b07100f0c.png">
> 
> Can you develop your ideas a little bit more please?

Sure @arojascastro! I was checking our analytics on its various options and realized that though we usually see that the US, India, Spain, Mexico, etc. are our main visitors, when we check the average duration of the site other countries stick out as well and will need to be taken into account. 

In this case, as far as May 2019, it looks like we have longer stay users in Namibia, Gabon, Togo, Liberia, Senegal, Niger, Somalia and Tanzania, in Africa and Guyana, Suriname, French Guiana and Haiti in South America and the Caribbean. I find it, at the least, very interesting since we never talk about them. 

@acrymble I don't mean to make it a fight on two proposals, but to refine the roles to make them more practical and successful (as I have done with my role of communication). As for this map, I see how our regional efforts are very much tied to language [maybe not dividing PH Globals yet as @drjwbaker says, ok] as Namibia (with German), Liberia, Somalia (though not their official language), and Tanzania speak/work in English, but Gabon, Senegal, Togo or Niger speak/work in French. In South America, Guyana's language is English, in Haiti and French Guiana they speak French and in Suriname Dutch. So in this case by doing PH North American and PH Latin America, we are leaving out all these countries as they are in South America but not Latin America as such. I know there is a lot of interest in DH in Haiti mainly through archives and post custodial initiatives with the US, why not contact them through PH en Français rather than @rivaquiroga? Resources and interest are to be taken into account as well as the channel of communication (language). My take on this is also due to the fact that we have a trilingual journal in place already - if it were only in one language, yes, I would just focus on each region as I do for a project I'm working on that is looking into the challenges in the Caribbean, in three languages.  

Maybe I'm overthinking it but I want to me mindful. Maybe it's just a matter of having a page where we explain the current target actions of each role, such as PH Global Africa is currently working towards a collaboration with partners in Senegal to extend our PH in French community?

**Comment by acrymble** [comment]
I think we might be talking past one another. I think one of the reasons for that is my choice of the term "Latin America", which having grown up far away from Spanish speaking parts of the world means to me "Everything south of Texas". I realize that's not accurate, and I think that it may be confusing things, judging by your points about Haiti and Guyana. What I should have said is "Central and South America, and the Carribbean" rather than "Latin America"

We already have linguistic-focused people. Each managing editor (should in my view) be actively engaging with their linguistic communities of potential authors and contributors. That's what a good editor does. They promote their publication with their communities.

What we don't have is a strategy for ensuring we are geographically representative, or even geographically aware.

We also have to be very careful about what seems to be the suggestion that we should carve Africa back up into its colonial languages. A language-based approach means we will never have anyone thinking about places where people don't speak the languages of our publications.

So I see this as a strategy that is in addition to an (already established) linguistic strategy. That's why I'm still struggling to see what the concern is.

**Comment by acrymble** [comment]
And as I said, I don't really mind what geographical divisions we use. If we want someone for each of South America, Central America, and the Caribbean, that's fine with me. We also don't need to fill them all right away, nor do we have to assume they'll be filled by current team members. It can be aspirational.

**Comment by drjwbaker** [comment]
> We also have to be very careful about what seems to be the suggestion that we should carve Africa back up into its colonial languages.

My bad. I didn't mean to imply that.

**Comment by JoshuaGOB** [comment]
Hi all. This is a vital discussion and one that probably doesn't have an ideal solution. Nevertheless, I would like to suggest some things that have arisen from the work that my institution does outside of the US with digital scholarship and projects. 

Geographic divisions definitely make sense when we have a lack of personnel to attend to the needs of a wide area, be it because of the amount of people or linguistic diversity. A person dedicated to Latin America, for example, will probably not be able to communicate with all of the region, but they could potentially recognize which languages are needed and can then recruit current or future members to assist in outreach. This is not to imply that language completely determines interest or skill level, it's just that it's essential at least at the point of outreach. It's also important to point out here that this is an extra task that would not have to be performed by other regions that have more well-developed DH communities and less diverse contexts that complicate participation in PH. I think that workload needs to be taken into account if we are creating geographic regions and that may be a more substantial obstacle than time zones or distance.  

Maybe a hybrid approach between linguistic and geographic divisions would be useful considering the limitations and advantages that each produce. Considering that we have English, French, and Spanish speaking members, it makes sense to have point people for those languages within the geographic regions and that those roles are as clearly defined as the larger continental titles proposed.

**Comment by Gimena** [comment]
I'd suggest having local chapters in difefrent counties with a coordinator. The concept of region is, from my point of view, too loose.

**Comment by quinnanya** [comment]
I'd think that outreach and coordination is going to happen most effectively in a local way -- incorporating lessons into events held at an institution or in a metro area, soliciting them from workshop instructors (and many workshops are similarly held at scales equal to or smaller than a country). I think @Gimena has a good idea in making it national, which would also address the issues raised by @arojascastro about not being up to date about what's going on in Norway or Russia. It might make sense to organize those country-level coordinators by region, if only as a proxy for time zone to facilitate meetings, though that doesn't always work out nicely. But I can imagine country-level coordinators providing a useful level of granularity to the outreach -- with the expectation that some country coordinators will be more active than others, and the first group of coordinators will play a role in finding coordinators in other countries, particularly where trans-national collaborations are already underway.

**Comment by acrymble** [comment]
Thanks for the contributions everyone. Personally, I'd love to have a person in every country! I think we'd need to raise some money to coordinate that though. And by some, I mean a lot.

**Comment by tillgrallert** [comment]
Thank you so much for having this important discussion and thank you @quinnanya for sharing parts of it on twitter. There seem to be practical and theory-driven views on how to map engagement with the world outside the Global North (however, ill-suited the term might be). In addition to geographic regions and linguistic communities, there is the issue of script and writing systems and how they again translate into regions and communities of practice  that has not yet been mentioned. Currently, PH is safely in latin-script territory with all the advantages that come with it (character encoding, programming languages and tools tailored to this script). As far as I can see, the societies and communities of people relying on other writing systems and the various problems that arise from trying to computationally approach cultural artifacts using these writing systems, cannot be addressed with the proposed structure. 
If PH was to further diversify into additional languages, it would make a lot of sense to start by addressing a (virtual) community of all those working on and in non-Latin scripts --- for instance, many of these communities will not be able to make use of any distant reading approach because they have no corpora to work with and no access to functional OCR technology for their particular combination of script and language. It might seem counterintuitive to pool engagement with communities in Central Asia, Indonesia, the Middle East and West Africa, but in terms of the affordances under which any DH approach must operate, they might see a lot of common ground before the need to diversify into smaller and localised communities.

**Comment by arojascastro** [comment]
Thank you all for your contributions. We have now a richer view of the problem. I think the ideal approach would be local as @Gimena suggested, but we do not have enough editors to act in each scenario and our team is already big enough so our monthly meetings are a kind of problem. 

I also agree with @tillgrallert about Latin script limitations, and I think we are not covering problems derived by non-Latin scripts with our tutorials, but I do not see either how can we change the situation if we do not recruit people from Arab countries, or we do not promote the writing of tutorials that deal with these problems. 

After reading other's views, I have a stronger opinion about this, and I am convinced now that we should face these challenges as linguistic teams coupled with our editions for one practical reason: our publishing structure and our human resources are limited, and we cannot give support to more language initiatives neither cover some aforementioned problems. Again, I think there is plenty of room for the English team to recruit people from Australia, India or South Africa and organize an event there; or for the French team to engage with French Africa. Otherwise, it is an unrealistic endeavour since this is a volunteer initiative. 

Perhaps we should stop using the word *global* because we will never be global, that's quite impossible, we can be a transnational or international project, but global... I do no think so. I would also advice to define concrete actions for 2019 and implement them rather than trying to split the world into regions, languages or communities.

**Comment by tillgrallert** [comment]
>I also agree with @tillgrallert about Latin script limitations, and I think we are not covering problems derived by non-Latin scripts with our tutorials, but I do not see either how can we change the situation if we do not recruit people from Arab countries, or we do not promote the writing of tutorials that deal with these problems.

I totally agree. The problem, in a way, is more a social one that cannot be addressed by pure "programming" solutions. Instead our non-Latin-script communities (for the lack of a better word) would need help, for instance in the form of tutorials, in building the necessary infrastructure (such as corpus building by scanning under dire circumstances, how to train ML algorithms for character recognition etc.).

**Comment by amsichani** [comment]
Thanks @tillgrallert for your useful points on a (somehow hidden) aspect of diversity in digital scholarship, that of non-Latin scripts languages, having impact both on existing (educational) resources, infra and tools. 
As a Greek with Arabic origins,  I 've thought several times about the challenges and at the same time the urgency of having a Greek or an Arabic version of PH, but this  kind of scaling up will require a different management of our resources at this stage.
On the other hand, I think the word "global" and "international" is there for a reason (in my view): because in PH we are keen to help making this kind of diversity issues visible, to discuss about obstacles, challenges and potential solutions.

**Comment by drjwbaker** [comment]
> Perhaps we should stop using the word *global* because we will never be global, that's quite impossible, we can be a transnational or international project, but global... I do no think so. I would also advice to define concrete actions for 2019 and implement them rather than trying to split the world into regions, languages or communities.

Depending on how you define global, or stats say we do reach a global audience. I'm for retaining 'Global' for reasons that build on but are different to @amsichani's comments: it is ambitious, and whilst our voluntary model does impose restrictions on what we can reasonably do, these feels like a moment to think ambitious.

**Comment by acrymble** [comment]
@arojascastro I just wanted to acknowledge your comments about the efficacy of our team size and how that affects group meetings. I know we all agree that's nothing to do with @amsichani and how she runs those meetings, but is instead a practical concern. So that we can address that, I will open a new ticket, as that's separate (but related) to the conversation we're having here.

It's interesting to me that we seem to have a difference of opinion that splits along linguistic lines, with the Spanish team seeming to prefer a linguistic approach and the English team broadly being more open to a regional (or some other non-linguistic) approach. Maybe that says something that we should consider.

**Comment by arojascastro** [comment]
Of course it’s has nothing to do with @amsichani - apologies it it sounded like a complaint. It actually wasn’t a complain. 

I would ask ourselves: 

- do we want to change our technological infrastructure? 

- do we want to expand our editorial board? 

Once we know the answer - let’s vote - then we should move to more ambitious tasks like this. But I would just relax, edit lessons and fix bugs. So my reply is no and no.

**Comment by acrymble** [comment]
@arojascastro I believe you've put your name down potentially to be the "European" person under the original model. But your post above makes it sound like that's no longer something you're interested in? We need to be clear that no one is obligated to take on a role or more roles than they want to. The roles should liberate you to make the type of contribution you want.

**Comment by arojascastro** [comment]
True @acrymble ! I’m not sure, I’m very conflicted... let me think about and I’ll update the page. I don’t want to commit to tasks I cant or want to develop. I just want to edit and fix bugs at the moment but at the same time I can’t avoid having an opinion on these issues... i guess I should just learn to shut up lol :p

**Comment by acrymble** [comment]
@arojascastro voices will always be welcome on all issues, I hope. You can still feed in your views on proposals and ideas without being the one who volunteers to take those ideas forward.

**Comment by rivaquiroga** [comment]
> It's interesting to me that we seem to have a difference of opinion that splits along linguistic lines, with the Spanish team seeming to prefer a linguistic approach and the English team broadly being more open to a regional (or some other non-linguistic) approach. Maybe that says something that we should consider.

Maybe this difference of opinion is precisely why the Spanish team prefers a linguistic approach. We are having this conversation in English. We are using the language of the English Team as _lingua franca_, not only here, but in almost every “visible” conversation in DH, technology, etc. 
I don’t think that the Spanish team is not open to a regional approach. Is just that language barriers are something we cannot ignore. We need to be aware that the regional approach may prevent people to join the conversation because of those barriers, and that for the moment we are not going to be able to reach them in the “global” way we dream. 

In the case of Latin America, the regional approach is something that makes sense, mainly because region and language almost overlap. Fostering the community that started in Bogotá and trying to extend it seems a reasonable first thing to do. But @jenniferisasi ‘s point is important. We can’t ignore that there are non-Spanish speaking countries, so we must have a strategy to reach them. And by “strategy” I mean just to have a plan, within our limited human & economical resources, to not leave them out of the conversation.

**Comment by drjwbaker** [comment]
@rivaquiroga Thank you for this vital contribution. My first thought was 'so what do you suggest we do'. But everything I know about '[Ally Skills](https://software-carpentry.org/blog/2018/02/valerie-aurora-keynote.html)', and with [Eddo-Lodge's book](https://www.bloomsbury.com/uk/why-im-no-longer-talking-to-white-people-about-race-9781408870570/) still in my head, I'm aware that asking individuals from historically marginalised groups to solve problems caused by the white hetronormative global north is not the answer.

So, I think at this stage that summarizing this thread might help. @acrymble has made a proposal to improve our internationalisation work. The team (note we are mostly Anglo-Spanish) has pushed back against it and suggested internationalising along linguistic lines. Despite some misgivings that this could divide the world up along colonial lines, this seems to be - intellectually - the direction we want to go.

Given the various qualifications and debates that got us to this position, it feels to me that the next thing we need to do is briefly write up a draft rationale, which includes pointing readers to this thread.

In the meantime to address the 'limited human & economical resources' we have, addressed by many of you, I'm hopeful that our work on giving us a financial/legal basis limited human & economical resources can give a platform to build up our resources.

**Comment by arojascastro** [comment]
In "localization studies" they usually talk about *locale*, that is the combination of a language and a territory. Here is some information: https://en.wikipedia.org/wiki/Language_localisation

## French


Tag | Locale
-- | --
fr-BE | Belgian French
fr-CH | "Swiss" French
fr-FR | Standard French (especially in France)
fr-CA | Canadian French

## Spanish

Tag | Locale
-- | --
es-ES | Castilian Spanish (as spoken in Central-Northern Spain)
es-MX | Mexican Spanish
es-AR | Argentine Spanish
es-CO | Colombian Spanish
es-CL | Chilean Spanish
es-US | American Spanish

## English 

Tag | Locale
-- | --
en-GB | British English
en-US | American English
en-CA | Canadian English
en-IN | Indian English
en-AU | Australian English
en-NZ | New Zealand English

Does this help? Or should we build a strategy to include people beyond our linguistic editions? I guess both strategies are compatible...

**Comment by drjwbaker** [comment]
Thanks @arojascastro. If we can fit with existing rationale like this, I'm happy. My worry with language orientated internationalisation only is that the expected 'outcome' of that work should be (and will be seen as) to build a group of editors to launch new language edition for each locale we internationalise into, rather than doing what is good for that locale (and us). If we can use existing framework to avoid that, again, I'm happy.

**Comment by arojascastro** [comment]
Very true @drjwbaker - I see your point. But let's suppose that we want to do something for people working in Abu Dhabi or any other speaking Arab country... unless we create a new edition in Arab, we will write and create content for them within the English edition, that is, using English as a *lingua franca*. I am happy with that, but then we have to be aware that the English edition is a *globalized* edition, thus editors from other locales should be recruited. Actually, that was my thinking when we started to discuss writing for a global audience...

**Comment by drjwbaker** [comment]
Exactly. Which is exactly the danger of us dividing our international work by language, as it could imply each language edition is for people who speak that language as a first language, whilst - as you say - EN written is for a global audience of second language speakers (which ES and FR are less likely to be, though will be for some communities in Central/South America and parts of Africa, right?). Which goes back to the way @acrymble carved up over internationalisation: by focusing on we can do for people in geographical areas regardless of the language(s) they speak....... :)

**Comment by JoshuaGOB** [comment]
The rationale that comes out of this discussion is going to be truly helpful here and for other projects that are attempting to navigate these issues. Thank you!

Just to add another consideration here. What you propose, @drjwbaker, raises the question for me of the implications of "focusing on we can do for people in geographical areas regardless of the language(s) they speak". For example, does that approach provide enough support for those linguistically heterogenous communities to create content? Or is the interest more focused on these communities "using" the site? @rivaquiroga point is really informative, most of the conversations in PH and DH are in English. This hasn't been in simplified English and writing the lessons for a "general audience" can be based on assumptions that I think this conversation will help to address, as I think @arojascastro is suggesting when talking about the need for recruiting that is representative of English locales.

**Comment by drjwbaker** [comment]
> For example, does that approach provide enough support for those linguistically heterogenous communities to create content? Or is the interest more focused on these communities "using" the site?

As the Bogota and https://github.com/programminghistorian/jekyll/issues/1179 examples show, at the moment 'provid[ing] enough support for those linguistically heterogenous communities to create content' requires partnering with those communities to secure external funding. If the work on finances https://github.com/programminghistorian/jekyll/issues/1301 is successful, I'd like to think we can start to divert our own funds to those activities, but that is an ambition (but - to repeat a comment from above - I think our PH Global work should aim to be ambitious). So, for now, I'd imagine this work will be 'focused on these communities "using" the site', but with three caveats: 1) we haven't got as far yet as defining roles (just how we divide the roles), 2) each PH Global person will - I imagine - have scope to develop their role as they go, and 3) we already see our 'users' as not just readers, but reviewers, writers, and - see above - people who raise issues and comment on issues.

**Comment by rivaquiroga** [comment]
I think a crucial point is to define what we expect as an 'outcome' of having Global Divisions. Right now, it seems that is to be able to reach people from different communities/contexts and see how, within the three languages we currently have, we can write content / run workshops / organize activities that are meaningful for them. Maybe, after a while, we realize through this process that a new PH edition in other language is needed. 
English is our _lingua franca_, and currently we can also use Spanish and French to (oficially) communicate with some communities. We can't expect right now to reach directly someone who only speaks Somali, for example. But what we can do is to reach someone who speaks Somali + English, French, or Spanish who can be the link, and who is interested in getting involved by, for example, running a local workshop in Somali based in a PH tutorial. Or by developing content that makes visible the needs of his/her community. And because those needs are probably similar to the ones of other communities, is not a bad idea to develop that content in a language that can be understood by a broader audience.

**Comment by drjwbaker** [comment]
We should probably remind ourselves of our core purpose here:

> We publish novice-friendly, peer-reviewed tutorials that help humanists learn a wide range of digital tools, techniques, and workflows to facilitate research and teaching. 

> Publicamos tutoriales revisados por pares dirigidos a humanistas que quieran aprender una amplia gama de herramientas digitales, técnicas computacionales y flujos de trabajo útiles para investigar y enseñar. 

> Nous publions des tutoriels évalués par des pairs qui permettent l'initiation à et l'apprentissage d'un large éventail d'outils numériques, de techniques et de flux de travail pour faciliter la recherche et l'enseignement en sciences humaines et sociales. 

We've done things other than this because members of the team have sought out and been granted financial support. Is a potential outcome of PH Global that one day we run 'a local workshop in Somali based in a PH tutorial'? Perhaps. I want to be ambitious, so I would like to think we could do something like that. 4 years ago the team - as it was then - probably hadn't imagined PH would launch an ES edition, run a workshop in Bogota, launch a FR edition, and be looking to run a workshop in West Africa. So, if we define the PH Global roles based on the ambition to, say, one day run 'a local workshop in Somali based in a PH tutorial', what things we handn't thought of might we not do that might have happened had we let the activity come from an engagement with a community?

So, maybe, we need a **broad** set of potential outcomes for PH Global and to let that - informed by all of the above - define the Global divisions we choose?

Can we define some specific tasks we need to do now to enable us to agree on how PH Global should proceed?

**Comment by acrymble** [comment]
Keeping @drjwbaker point about our mission in mind, the goal is to grow the audience of authors, reviewers, and readers in new and emerging markets. Good journals foster their communities and learn how to serve them better. That was the purpose of Bogotá and I think it’s done that

**Comment by drjwbaker** [comment]
I think we are in agreement Adam.

**Comment by drjwbaker** [comment]
> Can we define some specific tasks we need to do now to enable us to agree on how PH Global should proceed?

Can we?

**Comment by acrymble** [comment]
@drjwbaker I suggested above:

- Foster the community developed in Bogotá and encourage more submissions from participants, as many of these lessons are already half-written. Best achieved by someone Spanish speaking.
- Engage with West Africa or South Africa in a similar way, trying to identify and foster a community of potential contributors, while also learning about the needs in the region and helping us understand how best to meet those. This could be done in either French or English.
- Make connections with our user-base in India. Find out who they are and how we can serve them better. This should be done in English.

Is that what you mean?

**Comment by drjwbaker** [comment]
No (though these are great). I meant more, there is an impasse here on how to structure the PH Global work. I think we need specific tasks to break that impasse. Possible options include:

1. forming a group to make a fresh recommendation.
2. individuals above making alternative proposal to @acrymble's continent based approach, so that we can chose between two (or more) concrete alternatives.
3. accepting @acrymble's continent based approach for now, with an agreement to review in - say - 12 months.
4. dropping PH Global altogether.

**Comment by acrymble** [comment]
Why don't we try a hybrid approach and see what advantages and disadvantages each has? Then reflect in a year and revise as appropriate?

1. Spanish-speaking Western Hempisphere.
2. Canada & US (short-hand for "North America" in this context)
3. Africa
4. India

That gives us 2 macro-regional, 1 micro-regional, and 1 linguistic.

**Comment by drjwbaker** [comment]
> Why don't we try a hybrid approach and see what advantages and disadvantages each has? Then reflect in a year and revise as appropriate?

I like this. Though I have no preference on the initial divisions within PH Global.

**Comment by walshbr** [comment]
Note from #1286 - we thought it might make sense to wrangle a team of 3-5 folks committed to being on the global team to carry the conversation further and make decisions. In other words, rather than waiting until we have a decision on this issue to decide what the team will look like we thought it best to assemble the team and have this decision be their first task. #1285 lists these folks as interested in the global team - @amsichani @JMParr @acrymble @spapastamkou @rivaquiroga and @arojascastro (though recognizing Antonio that you said above you were still thinking about it)! So I'm going to open a  general ticket about the global team and ping you all in it. But any other interested folks can continue to participate here or self-nominate to be on the team itself.

**Comment by acrymble** [comment]
This is still an important conversation, but we have opened up a new ticket #1329 so I will close this one to archive the discussion.

**Comment by advsalahuddinahmed072** [comment]
[The All Pakistan Legal Decisions ](https://stlawrenceuniversity.on.worldcat.org/search?queryString=au%3D%22West%20Pakistan%20%28Pakistan%29%22&clusterResults=true&groupVariantRecords=true) #

**Comment by advsalahuddinahmed072** [comment]
Doctype:pdf
